x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_definition.tsx

```diff
@@ -83,6 +84,20 @@ export const RuleDefinition: React.FunctionComponent<RuleDefinitionProps> = ({
         : false)
     );
   }, [hideEditButton, canSaveRule, ruleTypeRegistry, rule]);
+
+  const ruleDescription = useMemo(() => {
+    if (ruleTypeRegistry.has(rule.ruleTypeId)) {
+      return ruleTypeRegistry.get(rule.ruleTypeId).description;
+    }
+    // TODO: Replace this generic description with proper SIEM rule descriptions
+    if (rule.consumer === AlertConsumers.SIEM) {
+      return i18n.translate('xpack.triggersActionsUI.ruleDetails.securityDetectionRule', {
+        defaultMessage: 'Security detection rule',
+      });
```

Comment:
LGTM as a temporary fix 👍 
Lets get the Security rules configured correctly in the RuleType Registry asap though - can you file an issue?

Reply:
Will do